Libraries/WebSocket/RCTWebSocketModule.m

```diff
@@ -48,6 +48,19 @@ - (void)dealloc
 RCT_EXPORT_METHOD(connect:(NSURL *)URL protocols:(NSArray *)protocols headers:(NSDictionary *)headers socketID:(nonnull NSNumber *)socketID)
 {
   NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:URL];
+  
+  // We load cookies from sharedHTTPCookieStorage (shared with XHR and
+  // fetch). To get HTTPS-only cookies for wss URLs, replace wss with https
```

Comment:
Maybe say "secure" cookies instead of HTTPS-only because "http-only" means something different for cookies